content/en/docs/concepts/services-networking/service.md

```diff
@@ -608,14 +608,22 @@ By default, for LoadBalancer type of Services, when there is more than one port
 ports must have the same protocol, and the protocol must be one which is supported
 by the cloud provider.
 
-If the feature gate `MixedProtocolLBService` is enabled for the kube-apiserver it is allowed to use different protocols when there is more than one port defined. 
+If the feature gate `MixedProtocolLBService` is enabled for the kube-apiserver it is allowed to use different protocols when there is more than one port defined.
 
 {{< note >}}
 
 The set of protocols that can be used for LoadBalancer type of Services is still defined by the cloud provider.
 
 {{< /note >}}
 
+#### Disabling Load Balancer Node Ports
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+Starting in v1.20, you can optionally disable node port allocation for a Service Type=LoadBalancer by setting the field `spec.allocateLoadBalancerNodePorts` to `false`.
```

Comment:
Is this always possible? If a cluster admin doesn't want this alpha behavior, can they opt out cluster wide?

Reply:
> Is this always possible? If a cluster admin doesn't want this alpha behavior, can they opt out cluster wide?

For alpha yes they can opt out using the feature gate. When we promote the feature to Beta/GA, any user (they can write to the Service) can toggle the API field however they want to enable or disable node ports. By default the node port is always on for compatibility reasons. 

If a cluster admin wanted to add some policy around this field (i.e. always allocate node ports or always de-allocate node ports) I think they would have to write an admission webhook or use a policy agent. We don't plan to add anything in apiserver to enforce the values of this field, but happy to discuss this further if we feel users might want this level of control.

Reply:
Interestingly when we first discussed the KEP, we started with global apiserver flag for this behavior, but we later moved to api fields instead https://github.com/kubernetes/enhancements/pull/1514#discussion_r379190691

Reply:
> For alpha yes they can opt out using the feature gate.

Let's document which feature gate this is.

Reply:
ack, will add this to the feature gate doc and mentio nthe feature gate here.